estate/models/inherited_user_model.py

```diff
@@ -0,0 +1,6 @@
+from odoo import models, fields
+
+class InheritedUserModel(models.Model):
+    _inherit = 'res.users'
+
+    property_ids = fields.One2many('estate.property', 'salesperson_id')
```

Comment:
Try to include an EOL at the end of the last line so that when future commits will append content to the file, this last line will not be impacted.

Reply:
- [ ]